hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/legacy/core/jproto/TxnReceipt.java

```diff
@@ -380,8 +380,11 @@ public static class Builder {
         private long newTotalSupply;
         private TxnId scheduledTxnId;
         private long[] serialNumbers;
+        private boolean isRevertable = true;
```

Comment:
Need javadocsin which cases this flag is set to true and false.

Reply:
Added!